### ISSUE TITLE
Ontology no longer builds due to unsatisfiable classes in the ontology

### ISSUE THREAD

**Issue by kltm** [geneontology/go-ontology#23431]
From about May 30th at midnight, the full ontology build fails with the following error:
```
09:38:23  ROBOT_JAVA_ARGS=-Xmx48G robot --catalog catalog-v001.xml merge -i extensions/go-lego-edit.ofn --collapse-import-closure true --include-annotations false reason -r ELK --exclude-duplicate-axioms true --remove-redundant-subclass-axioms true --exclude-tautologies structural relax reduce --named-classes-only true annotate -O http://purl.obolibrary.org/obo/go/extensions/go-lego.owl -V http://purl.obolibrary.org/obo/go/releases/2022-05-31/extensions/go-lego.owl -o extensions/go-lego.owl
09:39:44  ERROR There are 242 unsatisfiable classes in the ontology.
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1902060
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080166
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0030584
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075259
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0030582
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048658
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048657
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048530
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048656
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048655
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048654
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048653
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0140547
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010069
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010622
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010500
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010066
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0000909
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010065
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010068
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010067
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0000905
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010071
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048449
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048448
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1902040
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1990059
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1990058
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048443
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048442
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048441
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048440
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048447
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048446
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048445
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048444
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010072
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010080
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048439
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048317
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048438
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048559
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010081
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048316
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048437
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1902059
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1902058
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080126
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080127
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080128
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0097751
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075222
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075223
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075224
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075225
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010093
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048465
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048586
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048464
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048463
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080086
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048462
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048467
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048466
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048587
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048461
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048460
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010654
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0106154
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010095
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010094
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010097
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010096
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048459
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048454
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048575
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048453
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048574
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048452
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048573
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048451
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048458
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048579
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048457
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048578
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048456
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048577
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009793
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048455
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048576
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048450
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075280
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075281
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075282
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0090428
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070810
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080060
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080187
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075273
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075274
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075275
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075276
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048482
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075270
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048481
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075271
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048480
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075272
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010431
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075277
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075278
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075279
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070800
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070803
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070804
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070801
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070802
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080050
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070807
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048354
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070808
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070805
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070806
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048359
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0043582
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048479
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070809
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075262
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075263
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075264
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075265
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075260
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075261
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075266
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075267
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075268
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075269
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048825
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048826
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048700
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009901
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010214
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010451
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010450
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_2000034
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0031155
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_2000033
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009909
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009908
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1904095
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048498
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048497
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009910
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010227
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010229
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010344
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009911
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010588
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010582
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048281
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_2000014
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010234
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048283
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048282
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010231
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048836
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048835
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048834
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048833
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048838
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_2000028
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010480
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0031286
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0031287
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0090567
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070799
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070797
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070798
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0048508
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0060870
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080001
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009942
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0060865
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0060869
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009829
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0035670
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010254
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070788
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070789
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0120194
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0120195
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070787
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0120196
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075318
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075319
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1990068
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080112
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0080113
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_1902039
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075320
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009837
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009836
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009835
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070791
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070792
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070790
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070795
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075321
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070796
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075322
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010262
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070793
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075323
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0070794
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075324
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0120165
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0090380
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0120166
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0009960
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010154
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075314
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075315
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075316
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075317
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075310
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0090379
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075311
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0098755
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_2000693
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075312
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0090377
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_2000692
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0075313
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0090378
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0090376
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0062248
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0060863
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0060862
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0060861
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0060860
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0060858
09:39:44  ERROR     unsatisfiable: http://purl.obolibrary.org/obo/GO_0010162
09:39:44  
09:39:44  real	1m19.966s
09:39:44  user	4m32.348s
09:39:44  sys	0m27.080s
09:39:44  make: *** [Makefile:224: extensions/go-lego.owl] Error 1
```

**Comment by kltm** [comment]
Tagging @balhoff @pgaudet

**Comment by cmungall** [comment]
We need to run a step to get an explanation if there are unsatisfiable classes

https://robot.obolibrary.org/explain

**Comment by kltm** [comment]
Noting that this issue also prevents NEO from building.

**Comment by pgaudet** [comment]
will this get fixed by #23445?  

Thanks, Pascale

**Comment by kltm** [comment]
@pgaudet @cmungall It looks like we're building again as of midnight.
